coalib/settings/SectionFilling.py

```diff
@@ -28,6 +29,7 @@ def fill_settings(sections, acquire_settings, log_printer):
     global_bears = {}
 
     for section_name, section in sections.items():
+        section.aspects = extract_aspects_from_section(section)
```

Comment:
It's fine like this! Since `Section.__init__` also doesn't fill `.contents`. So it's also a kind of verification that the `Section` instance is finished. Otherwise you'll get an exception on accessing `section.aspects`